### DISCUSSION TITLE
Theming other UI components

### DISCUSSION THREAD

**Discussion by jeffgennari** [primer/github-vscode-theme#314] [Q&A]
Love the theme, thank you for making it. I was wondering if you've considered including other aspects of the UI in the themes, such as the `symbolIcon` or `debugTokenExpression`?

I know you can change these manually in your settings, but having it as part of the theme makes the UI look more consistent.

Thanks!

**Comment by simurai** [comment]
Ya, from [all the available theme colors](https://code.visualstudio.com/api/references/theme-color), there are probably quite a few that this theme doesn't override. Initially we just tried to cover some of the common ones, but there are still room for additions.
